apis/Google.Cloud.Speech.V1Beta1/Google.Cloud.Speech.V1Beta1.Snippets/SpeechClientSnippets.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Xunit;
+using static Google.Cloud.Speech.V1Beta1.RecognitionConfig.Types;
+
+namespace Google.Cloud.Speech.V1Beta1.Snippets
+{
+    public class SpeechClientSnippets
+    {
+        [Fact]
+        public void SyncRecognize()
+        {
+            var audio = LoadResourceAudio("speech.raw");
+            // Snippet: SyncRecognize
+            SpeechClient client = SpeechClient.Create();
+            var config = new RecognitionConfig { Encoding = AudioEncoding.Linear16, SampleRate = 16000 };
+            var response = client.SyncRecognize(config, audio);
```

Comment:
Definitely. Sorry, this evolved from my "let's just give it a go" attempt, and I should have fixed that up...